warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/handler/edge/ProtubufEdgeDeleteModeTest.java

```diff
@@ -462,7 +460,7 @@ public static void processEvent(Multimap<String,NormalizedContentInterface> even
                         log.info(keyString.trim());
                     }
                 }
-                Assert.fail(String.format("Expected: %s edge keys.\nFound: %s", expectedEdgeKeys, countMap.get(shardTableName), countMap.get(edgeTableName)));
+                Assert.fail(String.format("Expected: %s edge keys.\nFound: %s", expectedEdgeKeys, countMap.get(TableName.SHARD), countMap.get(edgeTableName)));
```

Comment:
Will this work to pass a String to a `Map<Text,Integer>`?

Reply:
Fixed in next commit.